src/components/plots/helpers/heatmap/vitessce/utils/generateVitessceHeatmapTracksData.js

```diff
@@ -0,0 +1,27 @@
+import getCellClassProperties from 'utils/cellSets/getCellClassProperties';
```

Comment:
Add comments explaining what is going on here and in other `populteHeatmap` related files